components/json/rs/src/lib.rs

```diff
@@ -329,3 +330,76 @@ pub fn convert_all_numbers_to_string(json: &str, path: &str) -> String {
         })
         .unwrap_or_else(|_| "".into())
 }
+
+/// Parses and re-serializes json with all numbers (`u64`/`i64`/`f64`),
+/// empty arrays [], `null` values in anarray
+/// or `null` valued properties removed, applied recursively
+/// at the specified path.
+///
+/// Non `null` and other than `u64`/`i64`/`f64` values are unchanged.
+/// The fields could be arbitrarily nested, as the conversion is applied
+/// to the entire JSON recursively. Returns an empty String if such
+/// conversion is not possible.
+///
```

Comment:
Why we need all these removals?

Reply:
it was mistake in description. I changed to:
```
/// Parses and re-serializes json with all numbers (`u64`/`i64`/`f64`),
/// converted to strings and empty arrays [], `null` values in array
/// or `null` valued properties removed, applied recursively
/// at the specified path.
```
we need to convert (`u64`/`i64`/`f64`) to string to have an ability to parse it like a string with IDL. 
Removals like `empty arrays [], `null` values in array or `null` valued properties` because IDL requires custom parser for values like `"propvalue": null`, It is more easy to remove such props at all and IDL will think it is `null`